src/lib/crypto/ecdh.c

```diff
@@ -225,14 +221,15 @@ pgp_ecdh_encrypt_pkcs5(rng_t *                  rng,
         return RNP_ERROR_NOT_SUPPORTED;
     }
 
-    if (*wrapped_key_len < ECDH_WRAPPED_KEY_SIZE) {
+    // +8 because of AES-wrap adds 8 bytes
+    if (*wrapped_key_len < (m_padded_len + 8)) {
         return RNP_ERROR_SHORT_BUFFER;
     }
 
     // See 13.5 of RFC 4880 for definition of other_info_size
     const size_t other_info_size = (pubkey->ec.curve == PGP_CURVE_NIST_P_256) ? 54 : 51;
```

Comment:
This was not changed, however RFC 4880bis defines this size as `The size of the KDF parameters sequence, defined above, is either 54 for the NIST curve P-256, 51 for the curves P-384 and P-521, or 56 for Curve25519.`.
Or we are not supposed to support ECDH with 25519 yet?

Reply:
Nop, that's part of RFC. RFC references RFC3394 

``   Compute C = AESKeyWrap( Z, m ) as per [RFC3394] ``
And that's what aes-3394 does
``
2.2.1 Key Wrap
...
   Outputs:     Ciphertext, (n+1) 64-bit values {C0, C1, ..., Cn}.
``

Reply:
We don't support djb's curve yet